java/ql/test/kotlin/library-tests/parameter-defaults/PrintAst.expected

```diff
@@ -0,0 +1,849 @@
+#   86|                   0: [TypeAccess] String
+#   86|                 1: [Parameter] y
+#   86|                   0: [TypeAccess] String
+#   86|                 2: [Parameter] z
+#   86|                   0: [TypeAccess] String
+#   86|               5: [BlockStmt] { ... }
+#   87|                 0: [ExprStmt] <Expr>;
+#   87|                   0: [MethodAccess] sink(...)
+#   87|                     -1: [TypeAccess] TestKt
+#   87|                     0: [VarAccess] y
+#   86|             3: [Method] f$default
+#   86|               3: [TypeAccess] Unit
+#-----|               4: (Parameters)
+#   86|                 0: [Parameter] p0
+#   86|                   0: [TypeAccess] String
+#   86|                 1: [Parameter] p1
+#   86|                   0: [TypeAccess] String
+#   86|                 2: [Parameter] p2
+#   86|                   0: [TypeAccess] String
+#   86|                 3: [Parameter] p3
+#   86|                   0: [TypeAccess] int
+#   86|                 4: [Parameter] p4
+#   86|                   0: [TypeAccess] Object
+#   86|               5: [BlockStmt] { ... }
+#   86|                 0: [IfStmt] if (...)
+#   86|                   0: [EQExpr] ... == ...
+#   86|                     0: [AndBitwiseExpr] ... & ...
+#   86|                       0: [IntegerLiteral] 2
+#   86|                       1: [VarAccess] p3
+#   86|                     1: [IntegerLiteral] 0
+#   86|                   1: [ExprStmt] <Expr>;
+#   86|                     0: [AssignExpr] ...=...
+#   86|                       0: [VarAccess] p1
+#   86|                       1: [VarAccess] p0
+#   86|                 1: [IfStmt] if (...)
+#   86|                   0: [EQExpr] ... == ...
+#   86|                     0: [AndBitwiseExpr] ... & ...
+#   86|                       0: [IntegerLiteral] 4
+#   86|                       1: [VarAccess] p3
+#   86|                     1: [IntegerLiteral] 0
+#   86|                   1: [ExprStmt] <Expr>;
+#   86|                     0: [AssignExpr] ...=...
+#   86|                       0: [VarAccess] p2
+#   86|                       1: [StringLiteral] hello world
+#   86|                 2: [ReturnStmt] return ...
+#   86|                   0: [MethodAccess] f(...)
+#   86|                     -1: [ClassInstanceExpr] new (...)
+#   86|                       -3: [TypeAccess] Object
+#   86|                     0: [VarAccess] p0
+#   86|                     1: [VarAccess] p1
+#   86|                     2: [VarAccess] p2
+#   90|         1: [LocalTypeDeclStmt] class ...
+#   90|           0: [LocalClass] 
+#   90|             1: [Constructor] 
+#   90|               5: [BlockStmt] { ... }
+#   90|                 0: [SuperConstructorInvocationStmt] super(...)
+#   90|             2: [Method] user
+#   90|               3: [TypeAccess] Unit
+#   90|               5: [BlockStmt] { ... }
+#   91|                 0: [ExprStmt] <Expr>;
+#   91|                   0: [MethodAccess] f$default(...)
```

Comment:
`f$default` is `static`, so it should have a type access as qualifier, instead of the below expression. Shouldn't this result in some missing expected flows?

Reply:
Surprisingly not -- I guess the rogue parameter -1 just gets ignored. I'll fix this though; thinking it should be a case of calling the static function without a `new` expression here, then calling the 'real' local function using the normal `(new WrapperClass()).f(...)` pattern in `f$default`.